modules/ROOT/pages/proxy-domain-support.adoc

```diff
@@ -15,16 +15,15 @@ API Manager enables you to automatically configure an API proxy to inherit an ex
 
 Note that API domains are not supported by Runtime Fabric or CloudHub support API domains. 
 
-*Important:* In an API domain, you may define several dependencies. Ensure that the versions of the dependencies defined in the domain have the same or later version than the ones defined in the API proxy that references the domain.
+*Important:* You may define several dependencies in an API domain. Ensure that the versions of the dependencies defined in the domain have the same or later version than the ones defined in the API proxy that references the domain.
```

Comment:
Is an API domain different from a proxy domain? The same (just for a proxy or an API)?